idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/service/PersistentSettings.kt

```diff
@@ -43,6 +44,7 @@ class PersistentSettings : SimplePersistentStateComponent<ValkyrieState>(Valkyri
         var addTrailingComma: Boolean by property(false)
 
         var showImageVectorPreview: Boolean by property(true)
+        var fileIndent: Int by property(DEFAULT_FILE_INDENT)
```

Comment:
I suggest leaving this property named `indentSize`, it would be easily distinguished with `indentStyle`.

Reply:
also I see this feature like this:

```var indentSize: Int by property(-1)``` 
`-1` will indicate that the user didn't select anything, and we should use the current IDE indent size. Otherwise, we should use the user-selected value.

Reply:
`fileIndent` can be renamed to `indentSize` across all changes in this pr

Reply:
done

Reply:
done